### ISSUE TITLE
API to fetch organization members

### ISSUE THREAD

**Issue by trundlr** [Facepunch/sbox-issues#7099]
### What can't you do?

Currently organization members are hidden by default. It would be great if we could authenticate with an api token and fetch a list of members that belong to an organization.

Our use case is validating specific actions against our custom backend - we want to accept requests that originate from a member of Ape Tavern, without having to hardcode steamid's of Ape Tavern members.

### How would you like it to work?

Pass an api key with a request to an endpoint that fetches organization information. If the api key has permission for a specific org, return more detailed information such as members (member permissions would be nice too!)

### What have you tried?

Hardcoding steam ids of Ape Tavern members.

**Comment by yuberee** [comment]
Personally I believe organization members should be public, there's no reason to have them private.

**Comment by jusvit** [comment]
> Personally I believe organization members should be public, there's no reason to have them private.

While transparency has its place, keeping organization members private serves important purposes. It allows the organization to speak with a unified voice and ensures focus remains on collective achievements rather than individuals. Private membership can protect individuals from unwanted attention while promoting team cohesion. It also gives members the freedom to contribute without external pressures, letting the organization's work and mission take center stage rather than individual personalities. When members change or move on, the organization's reputation and effectiveness remain stable, independent of specific individuals.

**Comment by Retroeer** [comment]
honestly yeah that's a decently convincing argument, if a game gets successful I'd imagine people would like the ability to opt out of getting essentially stalked

**Comment by yuberee** [comment]
If privacy is their goal then I assume they're also not going to have a Github org, a website, a Discord server, Steam group etc...

If someone wishes to remain anonymous they have the option to just not join the sbox.game org, you're not forced to since the only perk you receive is access to manage the uploads.

**Comment by Retroeer** [comment]
I imagine there will be more perks to being in the org later on, another one is being able to see error reports, soon we'll have user submitted bug reports and feature requests, don't see why someone would have to make their entire steam account private just to get away from it, its like if Discord forced you to link your steam account and you just couldn't toggle the visibility, it'd suck

**Comment by yuberee** [comment]
I'm not saying to expose their ass, they can have their privacy if they want but it should not be the default.

**Comment by Retroeer** [comment]
yeah i think an opt out would be fine, you'd still probably need the api key or whatever to fetch those who opt out
